tests-semanticdb/src/test/resources/metac.expect_2.13

```diff
@@ -1816,12 +1819,55 @@ Synthetics:
       b,
       c,
       d
-    ) == (1, 2, 3, 4))}).map[Tuple2[Tuple2[Int, Int], Tuple4[Int, Int, Int, Int]]]({(local8) => orig(e = (
+    ) == (1, 2, 3, 4))}).map[Tuple2[Tuple2[Int, Int], Tuple4[Int, Int, Int, Int]]]({(local8) => orig((
```

Comment:
looks like this logic had multiple redundant checks for `if (a, b, c, d) == (1, 2, 3, 4)` that got optimized away... is that how you understand it?

Reply:
I would guess so, but we don't really use those anyway.

Reply:
I think we could remove for comprehensions from synthetics, they are overly complex and unused.

Reply:
https://github.com/scalameta/scalameta/issues/4135